sites/all/themes/cetaviano/sass/main.scss

```diff
@@ -19,6 +19,7 @@
 @import "partials/patterns/buttons";
```

Comment:
Ya lo hice, pero al momento de hacer push, sale que todo esta al día

In sites/all/themes/cetaviano/sass/main.scss
<https://github.com/cetav-ddw/cetav-2015/pull/13#discussion_r47554953>:

> @@ -19,6 +19,7 @@
>  @import "partials/patterns/buttons";

te pasas a master, haces pull, te pasas a tu branch graduate_profile, haces
merge master, y despues push a graduate_profile

—
Reply to this email directly or view it on GitHub
<https://github.com/cetav-ddw/cetav-2015/pull/13/files#r47554953>.